examples/example-tls/README.md

```diff
@@ -35,65 +35,29 @@ USAGE: HelloWorldClientTls host port trustCertCollectionFilePath [clientCertChai
   Note: clientCertChainFilePath and clientPrivateKeyFilePath are only needed if mutual auth is desired.
```

Comment:
USAGE: HelloWorldClientTls host port _**[**_ trustCertCollectionFilePath [clientCertChainFilePath clientPrivateKeyFilePath] _**]**_

\+ Note: trustCertCollectionFilePath is not needed if you are using system default certificate.

Reply:
Done.